diff --git a/node_modules/capacitor-voice-recorder/ios/Plugin/CustomMediaRecorder.swift b/node_modules/capacitor-voice-recorder/ios/Plugin/CustomMediaRecorder.swift
index 79d79a4..82a1979 100644
--- a/node_modules/capacitor-voice-recorder/ios/Plugin/CustomMediaRecorder.swift
+++ b/node_modules/capacitor-voice-recorder/ios/Plugin/CustomMediaRecorder.swift
@@ -46,7 +46,7 @@ class CustomMediaRecorder {
             originalRecordingSessionCategory = recordingSession.category
             try recordingSession.setCategory(AVAudioSession.Category.playAndRecord)
             try recordingSession.setActive(true)
-            audioFilePath = getDirectoryToSaveAudioFile().appendingPathComponent("recording-\(Int(Date().timeIntervalSince1970 * 1000)).aac")
+            audioFilePath = getDirectoryToSaveAudioFile().appendingPathComponent("recording-\(Int(Date().timeIntervalSince1970 * 1000)).m4a")
             audioRecorder = try AVAudioRecorder(url: audioFilePath, settings: settings)
             audioRecorder.record()
             status = CurrentRecordingStatus.RECORDING
diff --git a/node_modules/capacitor-voice-recorder/ios/Plugin/VoiceRecorder.swift b/node_modules/capacitor-voice-recorder/ios/Plugin/VoiceRecorder.swift
index 36a24b8..5d96066 100644
--- a/node_modules/capacitor-voice-recorder/ios/Plugin/VoiceRecorder.swift
+++ b/node_modules/capacitor-voice-recorder/ios/Plugin/VoiceRecorder.swift
@@ -77,7 +77,7 @@ public class VoiceRecorder: CAPPlugin {
         let sendDataAsBase64 = customMediaRecorder?.options?.directory == nil
         let recordData = RecordData(
             recordDataBase64: sendDataAsBase64 ? readFileAsBase64(audioFileUrl) : nil,
-            mimeType: "audio/aac",
+            mimeType: "audio/mp4",
             msDuration: getMsDurationOfAudioFile(audioFileUrl),
             path: sendDataAsBase64 ? nil : path
         )
